cypress/support/step_definitions/fractalis-charts.js

```diff
@@ -17,21 +17,16 @@ then('There should be a box plot created', () => {
 
 then('I select two cohorts', () => {
   cy.get('span.ui-button-text.ui-unselectable-text').then(cohortsElms => {
-    if(cohortsElms && cohortsElms.length > 1) {
-      // if there is already more than one cohort, select the 2nd one
-      cohortsElms.eq(1).click();
-    } else {
-      // else creat and save a new cohort, then select it
-      const cohortName = 'temp_oracle_numerical';
-      cy.toggleNode('Public Studies');
-      cy.toggleNode('Oracle_1000_Patient');
-      cy.toggleNode('Numerical Variables');
-      cy.drag('numerical_11').drop(0);
-      cy.contains('Update cohort').click();
-      cy.get('#cohortName').type(cohortName);
-      cy.contains('Save cohort').click();
-      cy.get('gb-cohorts').contains(cohortName).eq(0).click();
-    }
+    // create a new cohort
```

Comment:
why changing this logic?

Reply:
this test is failing:

> Create a box-plot with a numerical variable from the Category View with multiple cohorts
 expected <g.fjs-box> to have a length of 2 but got 1

Reply:
hmm, strange, it's passing for me locally. 
I changed this logic, because it was selecting any cohort that was in the cohort panel, which does not have to be the expected one - with so many tests and when testing on the dev env there is a high chance there will be more cohorts. Creating it every time, when the test is run makes this test more independent.

Reply:
> there is a high chance there will be more cohorts.

This is what we want, right?

Reply:
yes, but the test expects to find 'numerical_11' variable in the cohort, which is not always the case (that is why it is failing currently).

Reply:
?? the test expects to see two boxplots regardless which cohorts are selected, as long as there are two cohorts, where is the numerical_11 part?

Reply:
https://github.com/thehyve/glowing-bear/blob/625d4bb975ab75aa62edf836c4b6f1c674e0bb7b/cypress/integration/fractalis-charts.feature#L14

Reply:
ah, I see, you are right, there is the variable dependency

Reply:
btw, the test is still failing:

> Create a box-plot with a numerical variable from the Category View with multiple cohorts
expected <g.fjs-box> to have a length of 2 but got 1

Reply:
local gb with transmart-dev and gb-backend-dev. Are you for sure up to date with the recent change from this branch? There is one commit that was added after the PR was created

Reply:
trying run with default cypress config?

> {
  "baseUrl": "https://glowingbear-dev.thehyve.net",
  "chromeWebSecurity": false,
  "video": false,
  "fixturesFolder": "cypress/fixtures/dev",
  "env": {
    "apiUrl": "https://transmart-dev.thehyve.net",
    "authentication-service-type": "oidc",
    "oidc-server-url": "https://keycloak-dwh-test.thehyve.net/auth/realms/transmart-dev/protocol/openid-connect",
    "oidc-client-id": "transmart-client",
    "gb-backend-url": "https://gb-backend-dev.thehyve.net"
  },
  "numTestsKeptInMemory": 3,
  "defaultCommandTimeout": 30000
}

Reply:
ah yes, but to fix this test there was a change added to the gb code (https://github.com/thehyve/glowing-bear/pull/541/commits/625d4bb975ab75aa62edf836c4b6f1c674e0bb7b), so the fix is only in this branch, "https://glowingbear-dev.thehyve.net" does not have

Reply:
right, I forgot about this part, thanks